src/main/java/com/fasterxml/jackson/core/base/ParserBase.java

```diff
@@ -403,6 +403,8 @@ public void overrideCurrentName(String name) {
             // 19-Jan-2018, tatu: as per [core#440] need to ensure no more data assumed available
             _inputPtr = Math.max(_inputPtr, _inputEnd);
             _closed = true;
+            // 30-May-2025, tatu: was missing before 2.20
+            _currToken = null;
```

Comment:
could this be viewed as a bug and potentially considered for 2.19.1?

Reply:
Ah. Test uses `ObjectReader.readValue(String)` which will create `JsonParser` but also close it (since caller wouldn't have access to it).

So it's probably ok -- can change test, but also explain why new behavior makes sense and how to avoid `JsonParser.close()` from occurring.

Reply:
Hmmmh. I think there's need for new `StreamReadFeature` flag for this, actually.